locales/ja/messages.json

```diff
@@ -1543,6 +1543,9 @@
     "auxiliaryAddRange": {
         "message": "範囲追加"
     },
+    "auxiliaryAddLink": {
+        "message": "結ぶ"
```

Comment:
Wow, I am impressed.
For next time, adding the translations (besides English) is not required in a pull request. We have a team of volunteer translators (hanging out in the #team_translation channel in Slack), and they use crowdin (https://crowdin.com/) to translate the messages, and the non-English message files are generated from there.

Reply:
Yes, they will be overwritten from there before the release of the next version. Sorry for the extra work ;)

Reply:
No worries. I've always loved linguistics and this was an enjoyable exercise, even if it does get overwritten. It was a welcome break from trying to translate/speak javascript. :)